tests/rules/sym_prop_merge.py

```diff
@@ -0,0 +1,20 @@
+def test1():
+    x = bar()
+    if cond:
+        y = 42
+    #ruleid: test
+    foo(x)
+
+def test2():
+    x = bar()
+    while cond:
+        y = 42
+    #todoruleid: test
```

Comment:
why this one is not handled?

Reply:
I haven't debug it to be 100% sure, but I believe it's related to Dataflow_svalue not making use of "_|_"  and instead considering that any variable for which we don't know anything it's a non-constant. That introduces some limitations with loops, there are also examples with const-prop. But left the `todoruleid` there to investigate/fix in the future.